src/App.js

```diff
@@ -1,27 +1,30 @@
 import React, { useState } from "react";
-import SettingsPage from './SettingsPage';
-import EditSettingsPage from './EditSettingsPage';
+import SettingsPage from "./SettingsPage";
+import EditSettingsPage from "./EditSettingsPage";
+import ErrorBoundary from "./ErrorBoundary";
 
 // hook for useState here
 // figure out what page to display
 function App() {
   const [currentPage, setPage] = useState({
-    page: 'list_settings'
+    page: "list_settings",
   });
 
-  if (currentPage.page === 'list_settings') {
-    return (
-      <div className="App">
-        <SettingsPage setPage={setPage} message={currentPage.message} />
-      </div>
-    )
-  } else if (currentPage.page === 'edit_setting') {
-    return (
-      <div className="App">
+  function getPage() {
```

Comment:
a function inside a function seems unnecessary. You can just assign the current page to a variable